diff --git a/setup.py b/setup.py
index d1092ca..c06dd85 100644
--- a/setup.py
+++ b/setup.py
@@ -5,7 +5,7 @@ from Cython.Distutils import build_ext
 ext_modules=[
     Extension("RplIcmp",
               ["RplIcmp.pyx", "tinyICMPlib.pxd"],
-              extra_link_args=["-lcap", "caplib.o", "icmplib.o"],
+              extra_link_args=["caplib.o", "icmplib.o", "-lcap"],
              )
 			      ]
 
@@ -14,4 +14,4 @@ setup(
   cmdclass = {"build_ext": build_ext},
   ext_modules = ext_modules,
 
-)
\ No newline at end of file
+)
